src/components/TextInput/BaseTextInput.js

```diff
@@ -20,14 +20,15 @@ class BaseTextInput extends Component {
         super(props);
 
         this.value = props.value || props.defaultValue || '';
-        const activeLabel = props.forceActiveLabel || this.value.length > 0;
+        const activeLabel = props.forceActiveLabel || this.value.length > 0 || props.prefixCharacter;
 
         this.state = {
             isFocused: false,
             labelTranslateY: new Animated.Value(activeLabel ? styleConst.ACTIVE_LABEL_TRANSLATE_Y : styleConst.INACTIVE_LABEL_TRANSLATE_Y),
             labelScale: new Animated.Value(activeLabel ? styleConst.ACTIVE_LABEL_SCALE : styleConst.INACTIVE_LABEL_SCALE),
             passwordHidden: props.secureTextEntry,
             textInputWidth: 0,
+            prefixWidth: 20,
```

Comment:
Should this be a constant?

Reply:
Hmm, actually `0` should be fine here.

Reply:
@parasharrajat why not `0`? 
When the prefix is more than one character, width `20` will be wrong.  And we're calculating the correct width `onLayout`